### DISCUSSION TITLE
Construcción del menu

### DISCUSSION THREAD

**Discussion by hrchioest** [TiendaNube/nimbus-design-system#125] [Q&A] [answered]
Estoy viendo [este issue](https://github.com/TiendaNube/issues/issues/24345) donde, desde el  menú al hacer click derecho no te da la opción para abrirlo en una nueva pestaña, ya que están construidos como si fuesen botones ¿Qué componente recomiendan que pueda usar ya que internamente tendría que ser como un elemento a, estuve revisando el componente MenuButton pero internamente tiene un button y un elemento p. Me dan ideas por favor? Desde ya gracias!

**Comment by jcampos89** [comment, marked as answer]
Hola @hrchioest! Gracias por su contribuición, de fato este comportamiento es un bug. Ya hemos convertido este discussion en una issue e vamos llevar a nuestro board de Jira, haremos todo lo posible para publicar el fix hasta el próximo release dia 28 de marzo.

**Reply by hrchioest** [reply]
Hola @jcampos89, genial! Muchas gracias!!
